hie.yaml

```
@@ -90,7 +90,7 @@ cradle:
     - path: "instrumentation/yesod/test"
       component: "hs-opentelemetry-instrumentation-yesod:test:hs-opentelemetry-instrumentation-yesod-test"
 
-    - path: "otlp/src"
+    - path: "otlp/./"
```

Can this be normalized?

```suggestion
    - path: "otlp"
```

---

This file was generated by `gen-hie`, so it could be changed but it's probably not necessary. It is likely that successive runs of `gen-hie` will continue to update it anyway.